src/core/twoFactor.ts

```
@@ -0,0 +1,418 @@
+  static get(name: string): EmailService | undefined {
+    return this.services.get(name);
+  }
+  
+  static getDefault(): EmailService | undefined {
+    return this.services.get('default');
+  }
+  
+  static list(): string[] {
+    return Array.from(this.services.keys());
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)